app/models/user.rb

```diff
@@ -4,8 +4,9 @@ class User < ActiveRecord::Base
 
   has_secure_password
 
-  has_many :reviews
+  has_many :reviews, -> { order("created_at DESC")}
```

Comment:
You are missing a space after `("created_at DESC") }`. Also you can use `order(created_at: :desc)` syntax :).